MyApplication/app/src/main/java/com/example/ivanbui/twittersearch/Status.java

```diff
@@ -9,7 +9,7 @@
 public class Status {
     @SerializedName("id_str")
     private String idStr;
-    @SerializedName("text")
+    @SerializedName("tweetMessageTextView")
```

Comment:
Yeah when you refactor you have to be careful for this. It does a string search and replace